tests/integ/test_dataframe.py

```diff
@@ -5222,3 +5222,17 @@ def test_create_dataframe_file_type(session, resources_path):
     assert csv_row["CONTENT_TYPE"] == "text/csv"
     image_row = json.loads(result[1][0])
     assert image_row["CONTENT_TYPE"] == "image/jpeg"
+
+
+@pytest.mark.skipif(not is_pandas_available, reason="Pandas is not available")
+def test_create_dataframe_empty_pandas_df(session):
```

Comment:
Is there a test where number of rows are zero but columns are non-zero to make sure that is allowed

Reply:
Yeah it is already in this test